### PR TITLE
DMOH-57: Fix serialization issue

### PR DESCRIPTION
**Pull request by zero2one**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+language: php
+
+php:
+  - 7.2
```
Comment (MPParsley) [inline .travis.yml]: 7.2 doesn't seem necessary any longer? it's in security support until 30 Nov 2020



```suggestion
```

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+matrix:
+  exclude:
+    - php: 7.4
+      env: TEST_SUITE=8.8.x
+    - php: 7.3
+      env: TEST_SUITE=8.8.x
+    - php: 7.2
+      env: TEST_SUITE=9.0.x
+    - php: 7.2
+      env: TEST_SUITE=9.1.x
```
Comment (MPParsley) [inline .travis.yml]: same remark
```suggestion
```

File: composer.json [hunk 3]
```diff
@@ -18,24 +18,19 @@
     "source": "https://github.com/StadGent/drupal_module_opening-hours"
   },
   "require": {
+    "php": "^7.2",
```
Comment (MPParsley) [inline composer.json]:
```suggestion
    "php": "^7.3",
```

File: grumphp.yml.dist [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        phpcs: ~
+        phpmd:
+            whitelist_patterns:
+                - /^src\/(.*)/
+            ruleset:
+                - phpmd.xml
+        phpunit:
+            config_file: "phpunit-coverage.xml"
+        phpversion:
+            project: '7.2'
```
Comment (MPParsley) [inline grumphp.yml.dist]:
```suggestion
            project: '7.3'
```

File: grumphp.yml.dist [hunk 5]
```diff
@@ -0,0 +1,30 @@
+parameters:
```
Comment (MPParsley) [inline grumphp.yml.dist:1]: or you could use digipolisgent/qa-drupal

File: phpcs.xml [hunk 6]
```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ruleset name="pbm_default">
-    <description>Default PHP CodeSniffer configuration for Organic Groups.</description>
```
Comment (MPParsley) [inline phpcs.xml:3]: :-)

File: src/Plugin/Field/FieldWidget/OpeningHoursWidget.php [hunk 7]
```diff
@@ -21,6 +20,9 @@
 /**
  * A widget bar.
  *
+ * @SuppressWarnings(PHPMD.ExcessiveClassComplexity)
```
Comment (MPParsley) [inline src/Plugin/Field/FieldWidget/OpeningHoursWidget.php:24]: too many dependencies?

**Review by MPParsley — APPROVED** [review]

File: grumphp.yml.dist (hunk 5 above)
Comment (zero2one) [reply grumphp.yml.dist:1]: not as long that the rules are not ok

File: src/Plugin/Field/FieldWidget/OpeningHoursWidget.php (hunk 7 above)
Comment (zero2one) [reply src/Plugin/Field/FieldWidget/OpeningHoursWidget.php:24]: Code refactoring for next version...

File: .travis.yml [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+install:
+  # Install Module.
+  - travis_retry composer install --no-interaction
+
+  # Install Drupal.
+  - travis_retry composer create-project drupal/recommended-project:$TEST_SUITE-dev "$DRUPAL_DIR" --no-interaction --dev
+
+  # Install Drupal Check.
+  - composer require mglaman/drupal-check --dev --working-dir="$DRUPAL_DIR"
```
Comment (MPParsley) [inline .travis.yml:42]: no longer needed (part of qa module now)

File: .travis.yml [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+  # Run GrumPHP checks.
+  - vendor/bin/grumphp run
+
+  # Test for deprecations.
+  - cd "$DRUPAL_DIR"
+  - phpenv config-rm xdebug.ini
+  - ./vendor/bin/drupal-check -ad "$MODULE_DIR"
+
+after_script:
+  - cd "$TRAVIS_BUILD_DIR"
```
Comment (MPParsley) [inline .travis.yml:58]: don't think this is still needed

**Review by MPParsley — APPROVED** [review]

**Pull request merged**